core/src/main/resources/jenkins/management/AdministrativeMonitorsDecorator/resources.css

```diff
@@ -112,3 +113,24 @@
     text-decoration: underline;
     text-decoration: var(--link-text-decoration--hover);
 }
+
+#visible-am-container #visible-am-list .alert-success a {
+    color: #155724;
+    color: var(--alert-success-text-color);
```

Comment:
I could stick a `!important` on the alert classes in style.less which would mean these changes aren't needed, but there's a very low risk it could break something, and it would block any customisation of the alerts

Reply:
(most of the css rules in this file are needed to undo the styles set by the page header)